source/include/signaling_data_types.h

```diff
@@ -0,0 +1,381 @@
+#ifndef SIGNALING_DATA_TYPES_H
+#define SIGNALING_DATA_TYPES_H
+
+/* Standard includes. */
+#include <stdint.h>
+#include <stddef.h>
+
+/*-----------------------------------------------------------*/
+
+/**
+ * Maximum number of URIs for an ICE server.
+ */
+#define SIGNALING_ICE_SERVER_MAX_URIS ( 4 )

```

Comment:
is this the list size max?

Reply:
This is maximum number of URIs for an ICE server and it is currently fixed. The current SDK seems to support one - https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/src/source/Ice/IceUtils.h#L56.

Reply:
Service response `IceServerList` should contain two items with `3 URIs` in each item.

Reply:
Hi @unicornss,
Thanks for your review! This is a nice one!

We refer to [IceConfigInfo](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h#L1396), the [MAX_ICE_CONFIG_URI_COUNT](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h#L615) in it is defined to 4. From the past log, I can only see 3 URIs in a single Ice server, but I thought there might be some reasons to define it to 4 in current SDK. We don't want to change any behavior comparing to current implementation. If it's sure that there are at most 3 URIs, I'm totally fine to update it.

Thank you.

Reply:
<img width="1169" alt="337520443-4994fdfb-37d0-4f4f-910b-201327c7f5a3" src="https://github.com/awslabs/amazon-kinesis-video-streams-signaling/assets/14988194/f81783e1-f8ba-4003-a05f-520737a8f12c">


https://docs.aws.amazon.com/kinesisvideostreams/latest/dg/API_signaling_IceServer.html#KinesisVideo-Type-signaling_IceServer-Uris

Doesn't seem to be any exact URL list length, just max 256 chars.